routes/messages.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const controllers = require('../controllers/messageController');

```

Comment:
You can use destructuring here, for example: `const  {getMessages, getOne, create, update, delete} = require('../controllers/messageController');`

That way our router will be cleaner:
```
router.get('/', getMessages)
router.get('/', getOne)
router.post('/', create)
```